site/docs/terms.md

```diff
@@ -20,39 +20,69 @@ title: "Terms"
 
 # Terms
 
-### Snapshot
+## Catalog
```

Comment:
I like this, the `catalog.md` was always a bit lonely as it was the only concepts page.

Reply:
I actually favor a little more on keeping `catalog.md` as a separate page, as the content is more than just terms. just keep this page in the new `Specification` section. Catalog is an important concept and deserves more details that the content page captured than simple term definition.

Reply:
agree that catalog is a term, which is good to be clarified in the `terms` page. but the content also talked a lot more about implementations. anyway, I can go either way on this one.

Reply:
nit: what about taking `Catalog` one level up? 

Currently its the first subtab under "Terms". Would it make sense to make it its on sub in "Specification"? 

![Screenshot 2025-02-18 at 9 06 17 AM](https://github.com/user-attachments/assets/536811b2-7cbf-4a97-a333-b7749f7e15a9)